crates/bevy_render/src/render_resource/pipeline_cache.rs

```diff
@@ -595,16 +605,41 @@ impl PipelineCache {
     pub fn queue_render_pipeline(
         &self,
         descriptor: RenderPipelineDescriptor,
+    ) -> CachedRenderPipelineId {
+        // SAFETY: We are setting `unchecked` to `false`, that's always safe
+        unsafe { self.queue_render_pipeline_ex(descriptor, false) }
+    }
+
+    /// Like [`Self::queue_render_pipeline()`], but disables runtime checks for the shader.
```

Comment:
Try to duplicate the docs here so readers don't need to follow through a link to get to the proper docs.